### ISSUE TITLE
ProjectionsManager is failing because there're no SSL certificate available.

### ISSUE THREAD

**Issue by AbdoDabbas** [charlessolar/Aggregates.NET#41]
Hi again :),
I'm trying to run the EventStore without enabling TLS, on my dev machine.
I faced and exception about SSL connection, and figured out (by EventStore's users help) that if I'm not using secure connection I need to specify the HTTP schema in the ProjectionsManager creation like this:  
```
var manager = new ProjectionsManager(client.Settings.Log, client.Settings.GossipSeeds[0].EndPoint, TimeSpan.FromSeconds(5), httpSchema: _httpSchema);
```
I cloned the source code of `Aggregates.NET` and did the changes needed and it worked !.
Two changes in file `EventStoreConsumer.cs`:
1. In method `EnableProjection` I replaced:
```C#
var manager = new ProjectionsManager(connection.Settings.Log, connection.Settings.GossipSeeds[0].EndPoint, TimeSpan.FromSeconds(30));
```
With:
```C#
var httpSchema = GetHttpSchema(connection.Settings);

var manager = new ProjectionsManager(connection.Settings.Log, connection.Settings.GossipSeeds[0].EndPoint, TimeSpan.FromSeconds(30), httpSchema: httpSchema);
```

2. In the method `CreateProjection` I did the same as the previous one.

3. Method `GetHttpSchema` is as the following:  
```C#
private string GetHttpSchema(ConnectionSettings settings)
{
	return settings.UseSslConnection ? "https" : "http";
}
```

What do you think? is that a correct solution for such an issue?

**Comment by AbdoDabbas** [comment]
I couldn't do a push to the branch I did the fix on, so waiting for your response.

**Comment by charlessolar** [comment]
@AbdoDabbas 
Ah I see, I wasnt aware the latest event store made this change.  

Did you want to submit a pull request?  I'll look into adding configuration settings to specify certs

**Comment by AbdoDabbas** [comment]
@charlessolar Yes actually I did the fix that I mentioned and created a private package for my team in our Azure Feed, it worked but I'm not sure if it's the correct way to do it, but it works :).
If it's the correct way I can do a pull request if you grant me access, because I tried to push the new branch I created and I wasn't able to.

**Comment by charlessolar** [comment]
Ah I see, you only need to commit to the clone you created https://github.com/AbdoDabbas/Aggregates.NET then you can create a pull request from those changes!

**Comment by AbdoDabbas** [comment]
Great, sorry this is my first time.

**Comment by AbdoDabbas** [comment]
@charlessolar I did a pull request.
